enthought/traitsui
# Update documentation for 7.1.1

Targeting [gh-pages]

These are the steps I followed for building the documentation of this bug fix release. Note that the steps are slightly different compared to a major/minor release.

```
git clean -ffxd
git checkout 7.1.1
python etstool.py install --toolkit=pyqt5
python etstool.py docs --toolkit=pyqt5
git checkout gh-pages
git pull
git checkout -b gh-pages-7.1.1
rm -rf 7.1
mkdir 7.1
rsync -r --exclude="\.buildinfo" docs/build/html/* 7.1/
rm -r `ls -d [![:digit:]]*`
cp -r 7.1/* .
```

To test:
```
python -m http.server
```
Checked the changelog is updated, tried the search.

There are no major changes to the 7.1 documentation apart from the changelog, which is expected. The rest are metadata change.

For #1419
